### PR TITLE
Add Prepare scanning screen

### LINKED ISSUE #14: Create Prepare scanning screen

### PR DESCRIPTION
**Pull request by marmontm**
This will close #14

### CODE REVIEW

**Comment by JoeNg93** [comment]
Can you give me some screenshot also?

**Comment by JoeNguyen93** [comment]
Seems that your screen can use my new `Header` component, please add it to your component, make a commit then `git push --force origin frontend-prepscan-screen`

File: front-end/package.json [hunk 1]
```diff
@@ -17,6 +17,12 @@
     "redux-thunk": "^2.2.0"
   },
   "devDependencies": {
+    "react-native-scripts": "1.11.1",
```
Comment (JoeNg93) [inline front-end/package.json]: It doesnt harm to make some change to `package.json` like this, but we actually dont need to do it

Comment (marmontm) [reply front-end/package.json]: For package.json is to add a script to start expo server with terminal with « npm start ». Then you just have to press « i » or « a » to start iOS or Android simulator. Please let it :)

Comment (JoeNguyen93) [reply front-end/package.json]: ok good finding 👍

Comment (marmontm) [reply front-end/package.json]: Thanks bro 👌

**Comment by marmontm** [comment]
Hey I add the Header to this screen. Now ready for PR :)

**Comment by marmontm** [comment]
Screenshot:   
screen shot 2018-03-24 at 16 34 45

**Comment by jeremydang** [comment]
Next time please prettify file before PR ;) Just find out this today when I look at your code.

**Comment by marmontm** [comment]
No problem ;)

**Pull request merged**